.github/workflows/kvrocks.yaml

```diff
@@ -325,6 +354,7 @@ jobs:
           ./x.py test cpp
 
       - name: Run Go Integration Cases
+        if: ${{ !matrix.riscv_toolchain }}
```

Comment:
Hi @wanghan-sanechips ,

We'd better support go integration tests for rsicv, the golang build itself could be an x86_Linux target.

Best Regards,
Edward